code/game/machinery/computer/communications.dm

```diff
@@ -365,6 +376,23 @@
 			SSjob.safe_code_timer_id = addtimer(CALLBACK(SSjob, /datum/controller/subsystem/job.proc/send_spare_id_safe_code, pod_location), 120 SECONDS, TIMER_UNIQUE | TIMER_STOPPABLE)
 			minor_announce("Due to staff shortages, your station has been approved for delivery of access codes to secure the Captain's Spare ID. Delivery via drop pod at [get_area(pod_location)]. ETA 120 seconds.")
 
+/obj/machinery/computer/communications/proc/emergency_access_cooldown(mob/user)
+	if(toggle_uses == toggle_max_uses)
+		to_chat(user, span_warning("This was your third free use without cooldown, you will not be able to use this again for [toggle_cooldown_seconds] seconds."))
```

Comment:
```suggestion
		to_chat(user, span_warning("This was your final free use without cooldown, you will not be able to trigger emergency access again for [toggle_cooldown_seconds] seconds."))
```

This means we can change the number 3 without having to update the text too, plus some minor changes.